packages/graphql-dynamodb-transformer/src/resources.ts

```diff
@@ -115,6 +115,16 @@ export class ResourceFactory {
         }
     }
 
+    public makeTableNameOutput(resourceId: string): Output {
```

Comment:
We need to make sure that this does not suffer the same issue as the ElasticSearchDomain output in situations where tables are hoisted to the root stack in migration situations.

Reply:
@mikeparisstuff you are right. Still working on the fix for #1047 then will apply it here if needed.

Reply:
Thanks